<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductCard</class>
 <widget class="QWidget" name="ProductCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="productImageLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: #82bfc7;
	color: black;
	border-radius: 4px;
	border:0px;
}</string>
   </property>
   <property name="text">
    <string>Image Preview</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="productNameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color:black;
	color: white;
	font-size: 10px;
	font-weight: bold;
	border: 0px;
	border-radius: 0px;
}</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="productPriceLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color:black;
	color: white;
	font-size: 10px;
	font-weight: bold;
	border: 0px;
	border-radius: 0px;
}</string>
   </property>
   <property name="text">
    <string>$ 2.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="productDescriptionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
	background-color: #82bfc7;
	color: #0b5572;
	font-size: 10px;
	font-weight: bold;
	border: 0px;
	border-radius: 0px;
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #82bfc7;
	font-weight: bold;
	border-radius: 5px;
	border: 0px;
}
QPushButton:hover {
	background-color: #0b5572;
	color: white;
}</string>
   </property>
   <property name="text">
    <string>&lt;button&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>201</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
	background-color: black;
	border-radius: 3px;
}</string>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>productImageLabel</zorder>
  <zorder>productNameLabel</zorder>
  <zorder>productDescriptionLabel</zorder>
  <zorder>pushButton</zorder>
  <zorder>productPriceLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
